apps/transport/priv/repo/migrations/20190911085235_add_resource_sha.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Transport.Repo.Migrations.AddResourceSha do
+  use Ecto.Migration
+
+  def change do
+    alter table(:resource) do
+      add :sha256, :string
```

Comment:
shouldn't it be better to rename this variable (now that it can be something else than sha256) ?
`hash` ? `hash_id`? `unique_id` ?

Reply:
we can do this, which one do you prefer?

Reply:
hash_id ? or content_hash ?